tests/test_orient_symmetric.py

```diff
@@ -66,26 +66,25 @@ def testRelativeRotations(self, order):
         gs = rots_symm
         rots_gt = src.rots
 
-        nchoose2 = int(n_img * (n_img - 1) / 2)
-        errs = np.zeros(nchoose2)
-        diffs = np.zeros(order)
+        # Find the angular distance between each Rij and the ground truth.
         pairs = all_pairs(n_img)
+        angular_distance = np.zeros(len(pairs))
         for idx, (i, j) in enumerate(pairs):
             Rij = Rijs[idx]
             Rij_J = J_conjugate(Rij)
             Ri_gt = rots_gt[i]
             Rj_gt = rots_gt[j]
+            dist = np.zeros(order)
             for s in range(order):
                 Rij_s_gt = Ri_gt.T @ gs[s] @ Rj_gt
-                diffs[s] = np.minimum(norm(Rij - Rij_s_gt), norm(Rij_J - Rij_s_gt))
-            errs[idx] = np.min(diffs)
-        mse = np.mean(errs**2)
+                dist[s] = np.minimum(
+                    self.angle_dist(Rij, Rij_s_gt), self.angle_dist(Rij_J, Rij_s_gt)
+                )
+            angular_distance[idx] = np.min(dist)
+        mean_angular_distance = np.mean(angular_distance)
 
-        # Mean-squared-error is better for C3 than for C4.
-        if order == 3:
-            self.assertTrue(mse < 0.005)
-        else:
-            self.assertTrue(mse < 0.03)
+        # Assert that the mean_angular_distance is less than 5 degrees.
+        self.assertTrue(mean_angular_distance < 5)
```

Comment:
Should probably make this threshold the same everywhere, unless we have a good reason to put 5 here and 2 in the other.

Reply:
Changed to 5 in both.